packages/mdc-segmented-button/segment/component.ts

```diff
@@ -37,22 +40,41 @@ export type MDCSegmentedButtonSegmentFactory =
  * Implementation of MDCSegmentedButtonSegmentFoundation
  */
 export class MDCSegmentedButtonSegment extends MDCComponent<MDCSegmentedButtonSegmentFoundation> implements MDCRippleCapableSurface {
+  get ripple(): MDCRipple {
+    return this.rippleComponent;
+  }
+
   static attachTo(root: Element) {
     return new MDCSegmentedButtonSegment(root);
   }
 
   private index!: number; // assigned in setIndex by parent
   private isSingleSelect!: boolean; // assigned in setIsSingleSelect by parent
+  private rippleComponent!: MDCRipple; // assigned in initialize
   private handleClick!:
       SpecificEventListener<'click'>; // assigned in initialSyncWithDOM
 
+  initialize(
+    rippleFactory: MDCRippleFactory =
+      (el, foundation) => new MDCRipple(el, foundation)
+  ) {
+    const rippleAdapter: MDCRippleAdapter = {
+      ...MDCRipple.createAdapter(this),
```

Comment:
@ngwattcos is this going to be an issue with your adaptor changes?

Reply:
Just fyi, I used mdc-chips/chip as a reference for the ripple TS/SCSS code.

Reply:
@kseamon @gschrag Not that I'm aware of. I did remove the default adapter implementation because I assumed it would receive a class or object that fully implements MDCRippleAdapter.